cmd/bench-loopix-client/run.go

```diff
@@ -15,67 +15,62 @@
-	preGenerate := opts.Flags("--pregenerate").Label("PREGENERATE").Bool("Whether to pregenerate single packet to send it over and over again")
+	panic("The benchmark client is not yet adjusted to work with the remote directory server")
+	// opts := newOpts("run [OPTIONS]", usage)
+	// // port := opts.Flags("--port").Label("PORT").String("Port on which loopix-client listens", defaultBenchmarkClientPort)
+	// numMessages := opts.Flags("--num").Label("NUMMESSAGES").Int("Number of benchmark messages to send", 0)
+	// interval := opts.Flags("--interval").Label("INTERVAL").Duration("Minimum interval between messages to be sent", 0)
+	// preGenerate := opts.Flags("--pregenerate").Label("PREGENERATE").Bool("Whether to pregenerate single packet to send it over and over again")
+
+	// params := opts.Parse(args)
+	// if len(params) != 0 {
+	// 	opts.PrintUsage()
+	// 	os.Exit(1)
+	// }
+
+	// privC := sphinx.BytesToPrivateKey([]byte{66, 32, 162, 223, 15, 199, 170, 43, 68, 239, 37, 97, 73, 113, 106,
+	// 	176, 56, 244, 146, 107, 187, 145, 29, 206, 200, 133, 167, 250, 19, 255, 242, 127})
+	// pubC := sphinx.BytesToPublicKey([]byte{202, 54, 182, 74, 58, 128, 66, 117, 198, 114, 255, 254, 100, 155, 20,
+	// 	238, 234, 96, 62, 187, 68, 173, 114, 95, 131, 248, 227, 164, 221, 39, 43, 89})
 
-	params := opts.Parse(args)
-	if len(params) != 0 {
-		opts.PrintUsage()
-		os.Exit(1)
-	}
+	// cfg, err := clientConfig.DefaultConfig(defaultBenchmarkClientID)
+	// if err != nil {
+	// 	panic(err)
+	// }
 
-	privC := sphinx.BytesToPrivateKey([]byte{66, 32, 162, 223, 15, 199, 170, 43, 68, 239, 37, 97, 73, 113, 106,
-		176, 56, 244, 146, 107, 187, 145, 29, 206, 200, 133, 167, 250, 19, 255, 242, 127})
-	pubC := sphinx.BytesToPublicKey([]byte{202, 54, 182, 74, 58, 128, 66, 117, 198, 114, 255, 254, 100, 155, 20,
-		238, 234, 96, 62, 187, 68, 173, 114, 95, 131, 248, 227, 164, 221, 39, 43, 89})
+	// cfg.Debug.LoopCoverTrafficRate = 0.0
+	// cfg.Debug.DropCoverTrafficRate = 0.0
+	// cfg.Debug.FetchMessageRate = 0.0
+	// cfg.Debug.MessageSendingRate = 10000000.0
+	// cfg.Debug.RateCompliantCoverMessagesDisabled = true
 
-	client, err := client.NewClient(defaultBenchmarkClientID,
-		defaultBenchmarkClientHost,
-		*port,
-		privC,
-		pubC,
-		config.ClientConfig{},
-	)
-	if err != nil {
-		panic(err)
-	}
+	// client, err := client.NewTestClient(cfg, privC, pubC)
+	// if err != nil {
+	// 	panic(err)
+	// }
 
-	benchClient, err := benchclient.NewBenchClient(client, *numMessages, *interval, *preGenerate)
-	if err != nil {
-		panic(err)
-	}
+	// benchClient, err := benchclient.NewBenchClient(client, *numMessages, *interval, *preGenerate)
+	// if err != nil {
+	// 	panic(err)
+	// }
 
-	if err := benchClient.RunBench(); err != nil {
-		fmt.Fprintf(os.Stderr, "Failed to spawn client instance: %v\n", err)
-		os.Exit(-1)
-	}
+	// if err := benchClient.RunBench(); err != nil {
+	// 	fmt.Fprintf(os.Stderr, "Failed to spawn client instance: %v\n", err)
+	// 	os.Exit(-1)
+	// }
```

Comment:
Go nuclear on comments! :)

Reply:
Ah, I knew I forgot something - need to adapt the original benchmark client/provider to work with the new changes so can't go too nuclear on that code : )